addons/api/addon/models/worker.js

```diff
@@ -96,7 +98,6 @@ export default class WorkerModel extends GeneratedWorkerModel {
 
   @attr({
     description: 'The api tags set for the worker.\nOutput only.',
-    readOnly: true,
```

Comment:
non-blocking: im wondering why remove `readOnly: true` if this field can only be saved via the setApiTags method ?

Reply:
I removed it since it can be updated via the API. However, I am not sure if the way we update it changes that. Happy to change it back and/or continue discussing this.